client/export_test.go

```diff
@@ -147,6 +148,15 @@ func TestClient_ExportToTemplate(t *testing.T) {
 			),
 			wantErr: false,
 		},
+		{
+			name: "merged variable",
+			key:  "test/airmail/",
+			want: fmt.Sprintf("%s=%s\n%s=%s\n",
+				test.ShortKey4, test.KeyValueMap[test.Key4]+"?pool=10",
+				test.ShortKey5, test.KeyValueMap[test.Key5],
+			),
+			wantErr: false,
+		},
```

Comment:
Can we also test it with multiple variables case?

Reply:
Negative test case also would be recommended.

Reply:
you might want to add test cases with empty key, empty value, 2+ `:`